templates/contrib/jdbc/README.md

```diff
@@ -68,17 +68,17 @@ Driver version: 42.2.5
 quay.io/kiegroup/jboss-kie-postgresql-extension-openshift-image:42.2.5
 ```
 
-You can import these images into you OpenShift instance using the extension-image-streams.yaml with the following command:
+You can import these images into your OpenShift instance using the extension-image-streams.yaml with the following command:
 
 ```bash
 oc create -f extension-image-streams.yaml -m <NAME_SPACE>
 ```
 
 The namespace is the project that you are working on OpenShift, or, you can also choose a common namespace to install these imagestreams, like `openshift`
 
-## Building a extension image
+## Building an extension image
 
-All you need to do is install cekit2 and execute the `make build` command specifying the image you want to build, i.e.:
+All you need to do is install [CeKit](https://docs.cekit.io/en/latest/handbook/installation/instructions.html) and execute the `make build` command specifying the image you want to build, i.e.:
```

Comment:
I think there should be mentioned also cekit version. Kie Server is build with 3.2. With newer versions there is the issue with max depth reached for layers.

Reply:
In this case I don't think it really matters, lastest version is good enough.

Reply:
actually, there is a little inconpatibility, I fixed it as part of https://issues.redhat.com/browse/RHPAM-2930

Reply:
@jakubschwan I updated it https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/444
for this PR let's use cekit 3.2, later we can update it.